packages/apps/frontera/src/routes/src/components/CommandMenu/commands/GlobalHub.tsx

```diff
@@ -38,6 +38,15 @@ export const GlobalSharedCommands = observer(() => {
     store.ui.commandMenu.setOpen(false);
   };
 
+  document.addEventListener('keydown', (e) => {
```

Comment:
This event listener doesn't seem to have cleanup function, please put in in useEffect and add cleanup function on unmount